projects/cm_mcu/LocalTasks.c

```diff
@@ -30,6 +31,452 @@
 #include "common/log.h"
 #include "common/printf.h"
 
+// prototype of mutex'd print
+void Print(const char *str);
+
+// FIXME: the current_error_count never goes down, only goes up.
+void SuppressedPrint(const char *str, int *current_error_cnt, bool *logging, enum log_facility_t log_num)
```

Comment:
I propose we delete the SuppressedPrint everywhere

Reply:
Done.